README.md

```diff
@@ -1,2 +1,78 @@
 # v9-bears-team-02
-Add-project-description-here | Voyage-9 | https://chingu.io/
+## Members
+
+Joseph Chananga Myalla
+
+Lemboye Omolabake
+
+LearnToTeach | Voyage-9 | https://chingu.io/
+
+Web/Mobile application Open Source Application
+
+The project objective is to build teaching skills of any subject to students of all levels.
+The registered commuity of students will get encouragement, motivation which will help them in their learning journey. Parents, teachers mentors can guid and support community members (students)to learn and understand the material, the content delivery must be done by the students.
+
+## User Stories
+
+Apparantly there is alot of good courses, programs in a variet of digital media, with so much appreciation of the abundance of good learning resources, we want to create a community that can create learning material that is in real demand.
```

Comment:
Great work, just a little typo fixes ...` Apparently there is a lot of good courses, programs in a variety of digital media, with so much appreciation of the abundance of good learning resources, we want to create a community that can create learning material that is in real demand`